.eslintrc

```diff
@@ -21,6 +21,7 @@
     "sourceType": "module"
   },
   "rules": {
-    "valid-jsdoc": "off"
+    "valid-jsdoc": "off",
+    "prefer-promise-reject-errors": "off"
```

Comment:
Why add this?

Reply:
Chimee use Object instead of Error in Promise rejection, such as

https://github.com/Chimeejs/chimee/blob/c1d168b94e5f8e5e13495a37ec3f10c70c33b2bd/src/dispatcher/index.js#L273

and so on (Not just `src/dispatcher/index.js`). EsLint will report an error:

```shell
src/dispatcher/index.js

274:29  error    Expected the Promise rejection reason to be an Error                               prefer-promise-reject-errors
```
So, I thought the best way is to close `prefer-promise-reject-errors` rule.